includes/abstracts/abstract-wc-stripe-payment-gateway-voucher.php

```diff
@@ -282,7 +282,7 @@ public function process_payment( $order_id, $retry = true, $force_save_save = fa
 			];
 		} catch ( WC_Stripe_Exception $e ) {
 			wc_add_notice( $e->getLocalizedMessage(), 'error' );
-			WC_Stripe_Logger::log( 'Error: ' . $e->getMessage() );
+			WC_Stripe_Logger::error( 'Error: ' . $e->getMessage() );
```

Comment:
Note that in some cases, `$e->getMessage()` returns the serialised response object, so we may want to consider pushing the message to the context. This may also make sense as something to tackle in a follow-up PR, as it's addressing a slightly different issue.
```suggestion
			WC_Stripe_Logger::error( 'Exception processing voucher payment', [ 'exception_message' => $e->getMessage() ] );
```